src/components/WorkOrders/MobileWorkingWorkOrdersView/MobileWorkingWorkOrdersView.js

```diff
@@ -99,6 +132,32 @@ const MobileWorkingWorkOrdersView = ({ currentUser }) => {
       .slice(0, 1)
   }
 
+  const logWorkOrders = async (
+    operativeId,
+    ojaatEnabled,
+    inProgressWorkOrderIds,
+    startedWorkOrderIds,
+    currentWorkOrderId
+  ) => {
+    const body = {
+      operativeId,
+      ojaatEnabled,
+      inProgressWorkOrderIds,
+      startedWorkOrderIds,
+      currentWorkOrderId,
+    }
+
+    try {
+      await frontEndApiRequest({
+        method: 'post',
+        path: '/api/frontend-logging/operative-mobile-view-work-orders',
+        requestData: body,
+      })
+    } catch (error) {
+      // fail silently - user doesnt need to know if logging fails
```

Comment:
Could consider a console.error here just so we can investigate in the unlikely event of a failure